ompi/mpi/fortran/mpif-h/profile/Makefile.am

```diff
@@ -439,12 +437,12 @@ CLEANFILES += psizeof_f.f90
 # Build the MPI_SIZEOF code in a separate convenience library (see
 # lengthy comment in ompi/mpi/fortran/mpif-h/Makefile.am for an
 # explanation why).
-#if BUILD_FORTRAN_SIZEOF
+if BUILD_PMPI_FORTRAN_MPIFH_BINDINGS_LAYER
```

Comment:
Meh; that `if` shouldn't have been commented out.  Bad Jeff.  :frowning: 

But regardless, the same questions from above apply (i.e., perhaps we should make `BUILD_FORTRAN_SIZEOF` smarter...?).